addons/resource.language.en_gb/resources/strings.po

```diff
@@ -23453,3 +23453,9 @@ msgstr ""
 msgctxt "#39194"
 msgid "Enables advanced DXVA upscaler using NVIDIA \"RTX Video Super Resolution\" or \"Intel Video Super Resolution\".[CR]Used when video source is 1080p or less (progressive only) and source resolution is lower than display resolution.[CR]It's only available on specific hardware: NVIDIA RTX 40x, RTX 30x and Intel Arc A770, A750."
 msgstr ""
+
+#. Label for debug log component level Graphics - Direct3D, OpenGL or other platform low level API
```

Comment:
Please move to line 3161, currently:

#empty string with id 673

Reply:
Maybe they need to remain empty on purpose? Moot point anyway with reuse of VIDEO.

Reply:
If a new category is created it can be moved here, otherwise N/A.